### DISCUSSION TITLE
Dynmap port

### DISCUSSION THREAD

**Discussion by Polyana-Fontes** [PowerNukkit/PowerNukkit#1137] [Ideas]
Dynmap is cool, apparently there's no Nukkit port (https://github.com/webbukkit/dynmap/issues/2761) but they are accepting PRs (https://github.com/webbukkit/dynmap/issues/1877) maybe we could make a PR adding PowerNukkit support :)

**Comment by whes1015** [comment]
This is really great news, I think it can help many people.
